src/web/client/src/components/ProtectedRoute/ProtectedRoute.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { render } from "@testing-library/react";
+import { BrowserRouter as Router } from "react-router-dom";
+import ProtectedRoute from "./ProtectedRoute";
+
+describe("ProtectedRoute component", () => {
+  test("renders ProtectedRoute component", () => {
+    const { container } = render(
+      <Router>
+        <ProtectedRoute isLoggedIn={false} redirectPath={"/"} />
+      </Router>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  it("renders Redirect when isLoggedIn is false", () => {
+    const wrapper: any = shallow(
+      <ProtectedRoute isLoggedIn={false} redirectPath={"/"} />,
+    );
+    expect(wrapper.find("Redirect")).toHaveLength(1);
```

Comment:
Why are you comparing length? What is the exact path does it route to?

Reply:
See above